packages/edit-site/src/components/page-patterns/index.js

```diff
@@ -68,6 +68,8 @@ const defaultConfigPerViewType = {
 	[ LAYOUT_GRID ]: {
 		mediaField: 'preview',
 		primaryField: 'title',
+		displayAsBadgeFields: [ 'sync-status' ],
```

Comment:
I think we should declare a badge fields once and has nothing to do with `asColumnFields`. Let me refactor the code a bit..

Reply:
In terms of naming, I think we should remove the `displayAs` prefix everywhere. We already use `badgeFields` in the UI component below, and there's no difference from the existing `primaryField` and `mediaField` (all of this is about "marking" a set of fields and display them differently).

The same applies to `columnFields`, that can be changed in a different PR.

Reply:
Sounds fine to me.